azurerm/internal/services/storage/tests/resource_arm_hpc_cache_nfs_target_test.go

```diff
@@ -0,0 +1,333 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/storage/parsers"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMHPCCacheNFSTarget_basic(t *testing.T) {
```

Comment:
as almost everything is required we can probably get away with a basic test, and then update that goes basic -> basicUpdated -> basic

existing testing is alright thou, just a comment

Reply:
Yes, while the basic test could be there as a baseline, or maybe not be skipped when someday you want to run a subset among the repo by specifying the pattern `TestAccfoo_basic`.